config.schema.json

```diff
@@ -70,6 +70,18 @@
 				"title": "Motion Time",
 				"type": "number",
 				"default": 10000,
+				"description": "Time (in ms) for blinds to move completely up or down."
+			},
+			"motion_up_time": {
+				"title": "Motion Upward Time",
+				"type": "number",
+				"default": 10000,
+				"description": "Time (in ms) for blinds to move completely from down to up."
+			},
+			"motion_down_time": {
+				"title": "Motion Downward Time",
+				"type": "number",
+				"default": 10000,
```

Comment:
Do you feel like tackling the changes to make some of these optional etc.?  :)

For ex:
- https://github.com/oznu/homebridge-config-ui-x/blob/master/config.schema.json
- https://github.com/NorthernMan54/homebridge-alexa/blob/master/config.schema.json

We can then make a lot of properties "advanced" and configure the correct logic to give users the option to choose certain things.

Reply:
No problem. Will look into it.

Reply:
Thanks @slavikme. Moving this to #42